tox.ini

```diff
@@ -37,19 +38,26 @@ commands =
 
 [testenv:docs]
 basepython = python3
+skip_install = false
+# Until repoze.sphinx.autointerface supports Sphinx 4.x we cannot use it:
+deps = Sphinx < 4
 extras =
     docs
+commands_pre =
```

Comment:
What's this for?

Reply:
It ensures that a possibly inherited `command_pre` (from `[testenv]`) does not get executed. (`command_pre` is used in the `zope-product` template but the snipped for the documentation is shared between all templates.)